License.txt

```diff
@@ -6,7 +6,7 @@ Copyright (c) 2009 Sandia Corporation. Under the terms of
 Contract AC04-94AL85000 with Sandia Corporation, the U.S. Government
 retains certain rights in this software.
 
-Copyright (c) 2011-2021, Cantera Developers.
+Copyright (c) 2011-2022, Cantera Developers.
```

Comment:
There are lots of opinions on the web about this, most saying that leaving the second year off implies "to present". I think I'd like to do that to avoid having to update this every year.

Reply:
See
* https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated
* https://softwareengineering.stackexchange.com/q/130478
* https://softwareengineering.stackexchange.com/q/210472
* https://softwareengineering.stackexchange.com/q/227812

Reply:
NP. I skipped the second year in `License.txt`, but changed the verbiage to `2001-present` in Sphinx' `conf.py`.

Reply:
I'm unfamiliar with any precedent for "YYYY-present", and would rather stick to the example set by peers like NumPy ([docs](https://numpy.org/doc/stable/), [license](https://github.com/numpy/numpy/blob/main/LICENSE.txt)), Astropy ([docs](https://docs.astropy.org/en/stable/), [license](https://github.com/astropy/astropy/blob/main/LICENSE.rst)) or pandas ([docs](https://pandas.pydata.org/docs/), [license](https://github.com/pandas-dev/pandas/blob/main/LICENSE)), all of which use the year range notation in both locations. Or we can run this question by our friends at NumFOCUS who might be able to give an informed answer.

Reply:
As this discussion is completely tangential to the main scope of this PR, I am reverting to the original approach, i.e. the full year is now given as the end date in both instances. This can be revisited as needed.